scripts/to-graph.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ts-node
+// tslint:disable:no-console
+
+import * as fs from 'fs';
+import {legacy} from '../src';
+
+const pkgManager = process.argv[2];
+if (!pkgManager) {
+  console.error('requires package manager argument. e.g. npm');
+  console.log('usage: cat tree.json | npx ts-node to-graph.ts npm > graph.json');
+  process.exit(1);
+}
+
+async function go() {
+  const depTree = JSON.parse(fs.readFileSync(0).toString());
```

Comment:
it'd be nice to use constants like FD_STDIN=0, FD_STDOUT=1 for more obviousness.